src/network/network_admin.cpp

```diff
@@ -336,8 +335,15 @@ NetworkRecvStatus ServerNetworkAdminSocketHandler::SendCompanyInfo(const Company
 		p->Send_uint8(owner);
 	}
 
+
+	uint8 rating = std::count_if(Company::Iterate().begin(), Company::Iterate().end(), [&](auto company){
+	return c->old_economy[0].performance_history < company->old_economy[0].performance_history;});
```

Comment:
Coding style is a bit off here. I suggest something like this:

```
uint8 rating = std::count_if(Company::Iterate().begin(), Company::Iterate().end(), [&](auto company) {
    return c->old_economy[0].performance_history < company->old_economy[0].performance_history;
});
```

That way it is a bit more clear where the body of the lambda starts and ends :)